File: src/main/java/com/apmosys/employeeportal/dto/ProjectFilterDTO.java
package com.apmosys.employeeportal.dto;

import java.util.List;
import java.util.Map;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@ToString
@Getter
@Setter
public class ProjectFilterDTO {
	
	private String approvalStatus;
	private String completionStatus;
	private Long currentUserEmpId;
	private Boolean isAdmin;
	private Boolean isHod;
	private Boolean isOther;
	private List<Long> departmentsids;
	private List<GetDeptIdByRoleDTO> departments;
	
	private Integer days;

}

File: src/main/frontend/src/app/models/projectFilterDTO.ts
export class ProjectFilterDTO{
    approvalStatus: any = null;
    completionStatus: any = null;
    currentUserEmpId:any;
    isAdmin:boolean = false;
    isHod:boolean = false;
    isOther:boolean = false;
    departmentsids: any[] = [];
    departments:any[] = [];
  days: any;
  expiredProjectFilter: any;
  type : any;
}